bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerStorage.java

```diff
@@ -103,12 +104,19 @@ void initialize(ServerConfiguration conf,
     long addEntry(ByteBuf entry) throws IOException, BookieException;
 
     /**
-     * Read an entry from storage.
+     * Read an entry from storage. In case of {@link BookieProtocol#LAST_ADD_CONFIRMED} we are going to return the
+     * 'last' written entry.
+     *
+     * @throws Bookie.NoEntryException in case of missing entry
+     * @throws IOException in case of internal failure
+     *
+     * @return the entry, never null
      */
-    ByteBuf getEntry(long ledgerId, long entryId) throws IOException;
+    ByteBuf getEntry(long ledgerId, long entryId) throws IOException, Bookie.NoEntryException;
```

Comment:
what is the purpose of the changes in this file?

Reply:
Oh, I left the comment on the previous PR.
It is only documentation.
While digging into the internals I found it useful to add this docs in order not to need to run thru the implementations again

Reply:
reverted